src/io/flutter/run/daemon/DevToolsService.java

```diff
@@ -233,7 +234,7 @@ public void projectClosing(@NotNull Project project) {
   }
 
   private CompletableFuture<Boolean> pubActivateDevTools(FlutterSdk sdk) {
-    final FlutterCommand command = sdk.flutterPub(null, "global", "activate", "devtools");
+    final FlutterCommand command = sdk.flutterPub(null, "global", "activate", "devtools", "2.0.0-dev.1");
```

Comment:
Do we indent to stay fixed at this specific version (`2.0.0-dev.1`) for a while? If so, perhaps add a comment about why.